tests/playwright-tests/tests/public/seoMetadataFiles.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { test, expect } from '@playwright/test';
+import environment from '@util/env';
+
+const { PUBLIC_URL } = environment;
+
+test.describe('SEO Metadata Files', () => {
+  test('An xml sitemap file can be found at the expected route', async ({
```

Comment:
Since a lot of the logic relates to setting the URL and last modified date, I'd suggest checking that these values are displaying properly - not all of them of course, maybe just one from each of the main sections since they're handled independently?